offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -270,15 +289,60 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     return affiliationIds[0]
   }
 
+  private void setCustomerActive(int customerId, boolean active) {
+    String query = "UPDATE person SET active = ? WHERE id = ?";
```

Comment:
shouldn't this be fixed as 1 if you want to set a customer active? Otherwise, you can also set him as inactive. If this is intended then the name of the method should be changed.

Reply:
I thought I would provide functionality to also set a user active (for now we only need the inactive case). But I see that I named the method rather unintuitive...do you have a suggestion? Or should I just remove that possibility?